test/gtest/ucs/test_bitmap.cc

```diff
@@ -412,6 +412,43 @@ UCS_TEST_F(test_dynamic_bitmap, test_popcount_upto_index)
     EXPECT_EQ(1, popcount);
 }
 
+UCS_TEST_F(test_dynamic_bitmap, test_ffs_fns) {
```

Comment:
maybe consider adding tests for corner cases:
- bitmap of size 100, all bits are unset
- bitmap of size 100, all bits are set
- bitmap of size 0/1